wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -259,6 +260,16 @@ void foo(const ML::cumlHandle_impl& h, ...)
 }
 ```
 
+The example below shows one way to create `nStreams` number of internal cuda streams which can later be used by the algos inside cuML. For a full working example of how to use internal streams to schedule work on a single GPU, the reader is further referred to [this PR](https://github.com/rapidsai/cuml/pull/1015). In this PR, the internal streams inside `cumlHandle_impl` are used to schedule more work onto a GPU for Random Forest building.
```

Comment:
I plan to use this example in the threading update PR as well, as I've been waiting for this PR to get merged so that I can update it. IIRC, the RF PR specifically uses OMP to schedule the streams in multiple threads and I've captured an example of this in the threading updates to the developer guide already. 

Since this is a section on threading, I will probably combine the example with the one in that PR and also incorporate the more precise wording that Jiri & I have been working on in that PR.

Reply:
Makes sense. Thanks